kute-android-app/app/build.gradle

```diff
@@ -1,8 +1,23 @@
 apply plugin: 'com.android.application'
 
+<<<<<<< HEAD
```

Comment:
@NickHub4017 
yes sir it was a small confusion doing it within 10 to 15 minutes